src/components/Stepper/Stepper.js

```diff
@@ -163,17 +163,15 @@ function Stepper({ steps, onComplete, onCompleteActions }) {
     stepperStage === stepsCount &&
     stepState[stepperStage].status === STEP_SUCCESS
 
-  // Commented just in case we want to add again the auto close
-
-  // useEffect(() => {
-  //   let timeout
-  //   if (completed && !onCompleteActions) {
-  //     timeout = setTimeout(() => close(), 2000)
-  //   }
-  //   return () => {
-  //     clearTimeout(timeout)
-  //   }
-  // }, [close, completed, onCompleteActions])
+  useEffect(() => {
```

Comment:
Now that we have notifications support. Add back again the autoclose handler on every modal.